### DISCUSSION TITLE
Add support to ignore errors and continue even if specific release/chart failed when applying

### DISCUSSION THREAD

**Discussion by liofko** [helmfile/helmfile#844] [Ideas]
Hi,

For development porpose, we would like the feature of ignoring release/charts failure and continue the deployment of the other releases.
Setting such flag will help us:
- Identify all the failing deployments in one run (in actual deployment, not dry-run)
- Give us the ability to test releases which are not related to the failed ones

Thanks,

**Comment by yxxhero** [comment]
@liofko please post more details.

**Reply by liofko** [reply]
Hi @yxxhero
Thanks for your prompt response.

As I wrote, if we will be able to continue deployment even after some releases deployment failed with some flag (e.g. --ignore-errors) it will be easier to test/verify all the other MSs which deployed successfully.
Lets say I'm having helmfile with 10 releases, if chart 2 failed, the apply will continue with all the other releases.
Ofer

**Comment by jouve** [comment]
Something like [make -k](https://linux.die.net/man/1/make) ?

> -k, --keep-going
    Continue as much as possible after an error. While the target that failed, and those that depend on it, cannot be remade, the other dependencies of these targets can be processed all the same.

**Reply by liofko** [reply]
Exactly :-)
